odl/solvers/util/callback.py

```diff
@@ -854,6 +856,52 @@ def __repr__(self):
             self.logy)
 
 
+class CallbackPrintMemory(SolverCallback):
```

Comment:
It prints more than memory so perhaps some other name. `CallbackPrintHardwareUse` or something.